cmd/src/batch_exec.go

```diff
@@ -140,6 +147,15 @@ func executeBatchSpecInWorkspaces(ctx context.Context, flags *executorModeFlags)
 		}
 	}
 
+	// Grab the absolute path to the workspace files contents.
+	workspaceFilesDir := flags.workspaceFilesDir
+	if !filepath.IsAbs(workspaceFilesDir) {
+		workspaceFilesDir, err = filepath.Abs(workspaceFilesDir)
+		if err != nil {
+			return errors.Wrap(err, "getting absolute path for workspace files dir")
+		}
+	}
+
```

Comment:
I don't know that we need to do the `IsAbs` here; presumably `filepath.Abs(path)` on a path that's already absolute will just return the same thing anyway.